presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -168,6 +168,11 @@ public void close()
         }
     }
 
+    public boolean isAlive()
```

Comment:
This looks like a separate change. It would be nice to extract it into a separate commit and explain it in PR description.